### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by cphillips17**
Added updates to the admin page

### CODE REVIEW

File: src/views/Admin.js [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      <Layout>
+        <Affix offsetTop={60}>
+          <Sider width={250}>
+            <Menu
+              mode="inline"
+              theme="light"
+              defaultOpenKeys={['users', 'resources']}
+              style={{ height: '100%', borderRight: 0 }}
+            >
+              {/* <Menu.Item>
```
Comment (sshh12) [inline src/views/Admin.js:242]: Can you remove these sections of commented code

File: src/views/Admin.js [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+          </Sider>
+        </Affix>
+        <Content
+          style={{
+            padding: '24px 24px 24px',
+            marginTop: '10px',
+          }}
+          offsetTop={100}
+        >
+          <Card id="overview" style={{ marginBottom: '20px' }}>
```
Comment (sshh12) [inline src/views/Admin.js:275]: Can you turn theses Cards into their own subcomponents (you can keep them in the same file).

It'll make the JSX a bit more readable and make it the code just a bit more reusable.

File: src/views/Admin.js [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+
+// import antd - changed dependencies
+// import { Collapse } from 'antd';
+import { useHistory } from 'react-router';
+import Footer from '../components/Footer';
+import LoginButton from '../components/LoginButton';
+let TEMP_FRONTEND_ITEMS = [
+  { name: 'AI Design Assistant', logoURL: '/demo/aiglobal.png' },
+  { name: 'Fawkes', logoURL: '/demo/fawkes.png' },
+  { name: 'The A-Z of AI', logoURL: '/demo/theazlogo.png' },
```
Comment (sshh12) [inline src/views/Admin.js:45]: I'm not sure which one of these you are using, but go ahead and delete any unused imports here & probably TEMP_FRONTEND_ITEMS

**Review by sshh12 — CHANGES_REQUESTED** [review]
Look great, just some super small stuff.

**Comment by cphillips17** [comment]
Made updates and some changes - going to treat tabs on left as scrollable buttons to go to relevant section. I used raw values rather than refs to scroll, will update later.

**Review by sshh12 — APPROVED** [review]
LGTM

**Pull request merged**